imod/tests/test_mf6/test_mf6_simulation.py

```diff
@@ -461,3 +466,19 @@ def compare_submodel_partition_info(first: PartitionInfo, second: PartitionInfo)
     return (first.id == second.id) and np.array_equal(
         first.active_domain, second.active_domain
     )
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_import_from_imod5(imod5_dataset, tmp_path):
```

Comment:
Are we also going to add a test to see if it runs? Or do we assume our validation is good enough to capture all input mistakes, and save some precious CI time? I'm open to both options.

Reply:
I tried if it runs (it does), but I didn't add it to the test because that adds 20 seconds or so to the test run time.